exactonline/api/goodsdeliveries.py

```diff
@@ -0,0 +1,17 @@
+from .manager import Manager
+
+
+class GoodsDeliveries(Manager):
+    resource = 'bulk/Salesorder/GoodsDeliveries'
+
+    def filter(self, entry_id=None, **kwargs):
+        if 'select' not in kwargs:
+            kwargs['select'] = 'DeliveryAccount,DeliveryDate,DeliveryNumber,{}'.format(
+                'GoodsDeliveryLines/ItemCode,GoodsDeliveryLines/SerialNumbers/SerialNumber'
+            )
```

Comment:
```suggestion
            kwargs['select'] = (
                'DeliveryAccount,DeliveryDate,DeliveryNumber,'
                'GoodsDeliveryLines/ItemCode,GoodsDeliveryLines/SerialNumbers/SerialNumber'
            )
```

No format needed here, right?